README.md

```diff
@@ -208,6 +216,50 @@ If at least one `<expression>` does not fit onto the line, *all* lines are wrapp
     <expression>)
 ```
 
+## Loops
+
+Conditions in loops are formatted equally to conditions in if expressions.
+
+### While loops
+
+```Ada
+while <condition> loop
+   <statements>
+end loop;
+
+while
+   <condition>
+   and then <condition>
+loop
+   <statements>
+end loop;
```

Comment:
I was wondering if we could find some replacement for `<condition>` that make clear that this is a *basic* condition not connected by `and then` or `or else`. Ideas? (The same change should be applied to the `if` statements already in the document).

Reply:
In this case I'd rather define a generic single line and multi line condition, e.g.
```
<condition>

<multiline
 condition>
```
and then define how a singe line and multi line condition is formatted. We don't need to write a complex condition with and and or and so on each time as the formatting is always the same.

Reply:
The `<multiline\n condition>` looks odd. I see two issues with it: The way it is above could lead to the impression that follow-up lines are indented by one character More importantly, nothing would indicate that we put operators at the beginning of the line, which I find important.

Reply:
The multiline condition should only serve as placeholder, maybe it should be aligned. But the operator location etc. should be inside the definition of a multiline condition (which should then be somewhere else in the document).

Reply:
I see. What I dislike about a separate definition of `<condition>` and `<multiline condition>` is that the guide would loose its by-example style. So far, you could just skim over the examples and get a feeling for how to style your code. If we add non-intuitive definitions for conditions, this is not possible anymore.

For accessibility I'd rather prefer to find an intuitive alternative for conditions (like `basic condition`, `simple condition`) and leave the `and then`/`or else` connectives in the examples.

Reply:
Offline discussion:
 - use a default example condition for all cases where a multiline condition is used
 - add more complex examples for for loop headers (especially very long ranges)